tests/karate/src/test/java/karate-config.js

```diff
@@ -17,14 +17,14 @@ function fn() {
   if (env === 'go') {
     // customize
     config.host = '127.0.0.1';
-    config.port = 9000;
-    config.path = 'organizer/client';
+    config.port = 9001;
```

Comment:
Is it the Go server port 9001? Because I usually connect on 9000

Reply:
I changed this because Mariem said to receive the hearbeat messages you need to be connected to the server port. For the Scala be however, Kilian said it's enough to change /client to /server for this. We can test if this work and maybe get it more coherent between be1 and be2.

Reply:
Clear, thank you!